### DISCUSSION TITLE
So basicly it vs wont let me use the package manager

### DISCUSSION THREAD

**Discussion by ali-kingX3** [cefsharp/CefSharp#3985] [General]
[Package source] The V2 feed at 'https://www.nuget.org/Search()?$filter=IsLatestVersion&searchTerm='Cef'&targetFramework='net461'&includePrerelease=false&$skip=0&$top=26&semVerLevel=2.0.0' returned an unexpected status code '404 Not Found'.
![image](https://user-images.githubusercontent.com/96551835/151882564-d529d654-9132-4eb8-8b21-ebf35e5caa46.png)

**Comment by amaitland** [comment]
In future please use discussions to ask questions. This is not a bug in CefSharp. 

Did you try searching for something else? Likely a transient issue with Nuget.org (happens semi frequently) and is entirely out of my control (Microsoft owns Nuget.org).

**Reply by amaitland** [reply]
See https://www.nuget.org/profiles/amaitland for list of packages

**Reply by ali-kingX3** [reply]
![image](https://user-images.githubusercontent.com/96551835/151884253-8a6d3d58-2953-41e6-9016-210ee250518b.png)

**Reply by ali-kingX3** [reply]
![image](https://user-images.githubusercontent.com/96551835/151884486-be03bdb7-00ec-45a9-b27a-ce0b24fb3526.png)

**Reply by amaitland** [reply]
You can see on the website the package exists https://www.nuget.org/packages/CefSharp.WinForms/97.1.61

There is a download link you can test it and download via your browser. 

Try install another unrelated package. Basically I'd suggest trying again later.

**Reply by amaitland** [reply]
This is out of my control, try again later. If the problem persists try installing a different unrelated package for test purposes. Contact Nuget.org support.

**Reply by ali-kingX3** [reply]
ive tried manully installing now lol 


Severity	Code	Description	Project	File	Line	Suppression State
Error		Package 'CefSharp.WinForms 97.1.61' is not found in the following primary source(s): 'C:\Users\laure\Desktop\Some packages'. Please verify all your online package sources are available (OR) package id, version are specified correctly.

**Reply by ali-kingX3** [reply]
ive contacted the nuget website btw im gonna wait now
